pkg/volume/csi/csi_block.go

```diff
@@ -55,41 +59,35 @@ func (m *csiBlockMapper) GetGlobalMapPath(spec *volume.Spec) (string, error) {
 	return dir, nil
 }
 
+// getStagingPath returns a path (on the node) to a device file which will be symlinked to
+// Example: plugins/kubernetes.io/csi/volumeDevices/staging/{volumeID}
+func (m *csiBlockMapper) getStagingPath() string {
+	sanitizedSpecVolID := kstrings.EscapeQualifiedNameForDisk(m.specName)
+	return path.Join(m.plugin.host.GetVolumeDevicePluginDir(csiPluginName), "staging", sanitizedSpecVolID)
+}
+
+// getPublishPath returns a path (on the node) to a device file which will be symlinked to
```

Comment:
Please clarify the comment for func `getStagingPath` and `getPublishPath`.   They have the same comment.